### ISSUE TITLE
Is this code still maintained and improved?

### ISSUE THREAD

**Issue by p4u** [nucypher/goUmbral#24]
I would like to use it, but I'm not sure if it is still in development. The last commit is from september last year.

**Comment by hongyuefan** [comment]
> I would like to use it, but I'm not sure if it is still in development. The last commit is from september last year.

https://github.com/hongyuefan/prencrypt

**Comment by ckartik** [comment]
Yeah same, seems like https://github.com/nucypher/pyUmbral is the most up-to-date usage.

**Comment by derekpierre** [comment]
This repos is no longer maintained. 

The most up-to-date implementation of the Umbral scheme is https://github.com/nucypher/rust-umbral, which is a Rust implementation that provides bindings for javascript and python. These bindings are utilized in https://github.com/nucypher/nucypher-ts and https://github.com/nucypher/nucypher respectively.
